arangod/Cluster/ClusterInfo.cpp

```diff
@@ -561,6 +569,10 @@ void ClusterInfo::loadPlan() {
           VPackSlice const& viewsSlice = databasePairSlice.value;
 
           if (!viewsSlice.isObject()) {
+            LOG_TOPIC(ERR, Logger::AGENCY)
```

Comment:
if this is uncritical, why not downgrade the message to level WARN?

Reply:
Maybe even INFO. It is interesting to see this in the log and it should be rare, but the user does not have to do anything or to worry.